src/static/common/css/repo.css

```diff
@@ -19,6 +19,7 @@
 
 nav ul a {
     color: #616161 !important;
+    text-transform: capitalize;
```

Comment:
For custom CMS pages, this will force nav items like "Become a Reviewer" to be rendered as "Become A Reviewer".

I think its best to be more explicit here and use a filter on the specific nav item we want to fix which I believe is this one:
https://github.com/BirkbeckCTP/janeway/blob/19036ec3717d3440330f5fff20638bc712e780dd/src/themes/material/templates/repository/nav.html#L20

We can use `capfirst`:
https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#capfirst

Reply:
Fair enough, this fix isn't even part of #1738, so I'll remove it from this PR, it can be its own ticket and wait for a future PR.